src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -224,28 +224,35 @@ public string GetDialect(out string versionNumber)
         }
     }
 
+    /// <summary>
+    /// Information about the CV_CFL_LANG enumeration, 
+    /// which specifies the code language of the application or linked module in the debug interface access SDK.
+    /// https://docs.microsoft.com/en-us/visualstudio/debugger/debug-interface-access/cv-cfl-lang
+    /// </summary>
     public enum Language : uint
     {
         C = 0x00,
         Cxx = 0x01,
         FORTRAN = 0x02,
-        MASM = 0x03,
+        MASM = 0x03, // Microsoft Macro Assembler
         Pascal = 0x04,
         Basic = 0x05,
         COBOL = 0x06,
-        LINK = 0x07,
-        CVTRES = 0x08,
-        CVTPGD = 0x09,
+        LINK = 0x07, // linker-generated module
+        CVTRES = 0x08, // resource module converted with CVTRES tool
+        CVTPGD = 0x09, // POGO optimized module generated with CVTPGD tool
         CSharp = 0x0A,
         VisualBasic = 0x0B,
-        ILASM = 0x0C,
+        ILASM = 0x0C, // intermediate language assembly (that is, Common Language Runtime (CLR) assembly)
         Java = 0x0D,
         JScript = 0x0E,
-        MSIL = 0x0F,
+        MSIL = 0x0F, // Microsoft Intermediate Language (MSIL), possibly a result of using the /LTCG (Link-time Code Generation) switch
         HLSL = 0x10, // High Level Shader Language
         ObjectiveC = 0x11,
         ObjectiveCxx = 0x12,
         Swift = 0x13,
+        ALIASOBJ = 0x14, // module generated by the aliasobj tool
```

Comment:
>module [](http://example.com/codeflow?start=28&length=6)

What's the additional value of your comment?

Reply:
agree some of the comments does not add value at all, removed all to be consistent